lib/pbench/test/functional/server/test_put.py

```diff
@@ -249,3 +249,17 @@ def test_delete_all(server_client: PbenchServerClient, login_user):
             )
             assert response.ok, f"{response.text}"
             print(f"Deleted {t.name}")
+
+    @staticmethod
+    def test_user_survives_dataset_delete(
```

Comment:
I noticed here that your branch is lagging way behind main. One of the costs of a long-term branch is either keeping it up to date with periodic rebases against main or ... a really messy merge at the end. 😦

Reply:
yeah after this PR gets merged that's one of the things on my plate to get the `openid-connect` branch up to date with the `main`.